packages/page-accounts/src/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,606 @@
+// Copyright 2017-2020 @polkadot/app-accounts authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { BareProps } from '@polkadot/react-components/types';
+import { AccountId, Address } from '@polkadot/types/interfaces';
+
+import React from 'react';
+import { Label } from 'semantic-ui-react';
+import styled from 'styled-components';
+import { useAccountInfo, useApi, useRegistrars, useToggle } from '@polkadot/react-hooks';
+import polkascan from '@polkadot/apps-config/links/polkascan';
+
+import { classes } from '@polkadot/react-components/util';
+import { colorLink } from '@polkadot/react-components/styles/theme';
+import { AccountNameJudgement, AddressMini, AddressSmall, AvatarItem, Button, Icon, IconLink, Input, InputTags, Transfer } from '@polkadot/react-components';
+
+import { useTranslation } from '../translate';
+
+interface Props extends BareProps {
+  address: AccountId | Address | string | Uint8Array;
+  onClose: () => void;
+  onUpdateName: () => void;
+}
+
+const Screen = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 998;
+`;
+
+function Sidebar ({ address, className, onClose, onUpdateName, style }: Props): React.ReactElement<Props> | null{
+  const { t } = useTranslation();
+  const { api, systemChain } = useApi();
+  const { registrars } = useRegistrars();
+  const accountInfo = useAccountInfo(address);
+  const [isHoveringButton, toggleIsHoveringButton] = useToggle();
+  const [isTransferOpen, toggleIsTransferOpen] = useToggle();
+  const [isJudgementOpen, toggleIsJudgementOpen] = useToggle();
+
+  if (!accountInfo) {
+    return null;
+  }
+
+  const useIdentity = !!api.query.identity?.identityOf;
+
+  const _toggleTransfer = (): void => {
+    toggleIsTransferOpen();
+  };
+
+  const _toggleJudgement = (): void => {
+    toggleIsJudgementOpen();
+  };
+
+  const {
+    identity,
+    isCouncil,
+    isDevelopment,
+    isEditable,
+    isEditingName,
+    isEditingTags,
+    isExternal,
+    isInContacts,
+    isOwned,
+    isSociety,
+    isSudo,
+    isTechCommittee,
+    name,
+    onForgetAddress,
+    onSaveName,
+    onSaveTags,
+    setName,
+    setTags,
+    tags,
+    toggleIsEditingName,
+    toggleIsEditingTags
+  } = accountInfo;
+
+  const _onUpdateName = (): void => {
+    onSaveName();
+    onUpdateName && onUpdateName();
+  };
+
+  const hasFlags = [isDevelopment, isExternal, isSociety, isCouncil, isTechCommittee, isSudo].reduce(
+    (result: boolean, value: boolean): boolean => result || value,
+    false
+  );
+
+  const buttonProps = {
+    onMouseEnter: toggleIsHoveringButton,
+    onMouseLeave: toggleIsHoveringButton
+  };
+
+  return (
+    <>
+      <Screen onClick={onClose} />
```

Comment:
This actually "feels" wrong to me - so I click on an account, then click on the next and well, it doesn't "take". So I need to click-click to get the next one.

Add an explicit button to remove on the popup.

(Which really should take up the full sidebar, it is lost at the bottom atm.)

For instance, this works...

![image](https://user-images.githubusercontent.com/1424473/79341319-e5909300-7f2b-11ea-911c-d133a6f4fea4.png)